datamol/mol.py

```diff
@@ -1376,3 +1379,39 @@ def get_atom_positions(
         positions = positions[mapped_indices, :]
 
     return positions
+
+
+def remove_salts_solvents(
+    mol: Mol,
+    defnData: str = None,
+    defnFormat: str = "smarts",
+    dontRemoveEverything: bool = False,
```

Comment:
Usually datamol try to propose "python-like" args compared to rdkit by replacing the upper letter with `_{lower_case_latter}`: `dontRemoveEverything` -> `dont_remove_everything`